paasta_tools/cli/cmds/status.py

```diff
@@ -1743,6 +1745,134 @@ def print_tron_status(
     return 0
 
 
+def print_cassandra_status(
+    cluster: str,
+    service: str,
+    instance: str,
+    output: List[str],
+    cassandra_status,
+    verbose: int = 0,
+) -> int:
+    tab = "    "
+    indent = 1
+    output.append(indent*tab + "Cassandra cluster:")
+    indent += 1
+
+    status = cassandra_status.get("status")
+    state = status.get("state")
+
+    if state == "Running":
+        state = PaastaColors.green(state)
+    else:
+        state = PaastaColors.red(state)
+
+    output.append(indent*tab + "State: " + state)
+
+    output.append(indent*tab + "Nodes: ")
+    indent += 1
+    now = datetime.now(timezone.utc)
+    tableOkNodes = []
+    tableErrNodes = []
+
+    for node in status.get("nodes"):
+        ip = node.get("ip")
+        err = node.get("error")
+        inspectTime = datetime.strptime(node.get("inspectTime"), "%Y-%m-%dT%H:%M:%SZ").replace(tzinfo=timezone.utc)
+        details = node.get("details")
+
+        age = (now - inspectTime).total_seconds()
+        if age > 3600*24*365:
```

Comment:
we use `humanize` for this elsewhere in paasta

https://pypi.org/project/humanize/

Reply:
[Done](https://github.com/Yelp/paasta/compare/95ed3303271facb37d68254f527ceb0463d9c222..8dc6f14f14f4b12abf836a0fc158fc5152271a50)